app/views/layouts/application.html.erb

```diff
@@ -4,15 +4,17 @@
     <title>FastTracks</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```

Comment:
I like Bootstrap, it'll be good if we can get it customized to look unique. Time permitting of course.